propagation_binary.go

```diff
@@ -34,6 +36,13 @@ func (binaryPropagator) Inject(
 	if err != nil {
 		return err
 	}
+	if carrier, ok := opaqueCarrier.(io.Writer); ok {
+		buf := make([]byte, base64.StdEncoding.EncodedLen(len(data)))
+		base64.StdEncoding.Encode(buf, data)
```

Comment:
Mind adding a comment here to how you determined use `base64.StdEncoding`? I know that this as to inter-op with the tracers in other languages and I assume that they just all use base64 but it would be nice to have a reference to that decision somewhere.

Reply:
I just copied that from the other string / []byte cases...

Reply:
Ah, I see. I will do some more digging on my end. Please ignore the above comment. No action is needed.